var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -282,8 +282,8 @@ def config_settings(self, spec, prefix):
             "setup-args": {
                 # http://scipy.github.io/devdocs/building/blas_lapack.html
                 "-Dfortran_std": fortran_std,
-                "-Dblas": blas,
-                "-Dlapack": lapack,
+                "-Dblas": blas if blas != "Accelerate.framework" else "accelerate",
+                "-Dlapack": lapack if lapack != "Accelerate.framework" else "accelerate",
```

Comment:
Let's fix this in `py-numpy`'s `blas_lapack_pkg_config` instead.

Reply:
I don't know enough about the intended behavior/ontology of numpy's config to feel comfortable making that change; however, reading the scipy docs led me to believe that "accelerate" is treated specially by scipy, so this seemed like the right place to make that change anyway: https://docs.scipy.org/doc/scipy/building/blas_lapack.html#selecting-blas-and-lapack-libraries

Happy to accept a patch with different behavior as long as you can confirm scipy + numpy both build from source and pass their internal module-level Pytest suite with it.

Reply:
is pkgconfig even aware of system frameworks? when I was debugging the scipy build it was doing some crazy stuff where meson was falling back on scanning clang's configuration for framework paths, scanning those paths, and then somehow trying to invoke cmake (??!?) to find it and failing.

Reply:
Hmm, maybe @rgommers would know

Reply:
> How does numpy build against veclib?

The needed extra compile and link flags are part of the changes in the vendored `meson` inside numpy. 

> is pkgconfig even aware of system frameworks? 

It isn't, system frameworks are handled specially by Meson. `dependency('depname')` inside `meson.build` causes Meson to try different dependency providers in turn. If `depname` isn't specially handled, that's `pkg-config` then `CMake`. Other dependency providers can be added, for `'accelerate'` there's special handling in `numpy` but not `scipy` as of today.